packages/flutter/lib/src/material/card.dart

```diff
@@ -72,6 +72,7 @@ class Card extends StatelessWidget {
     this.color,
     this.elevation,
     this.shape,
+    this.borderOnForeground = true,
```

Comment:
assert that this property is non-null

Reply:
Done and added assert to `material.dart` as well